README.md

```diff
@@ -1 +1,22 @@
-# infrastructure
\ No newline at end of file
+# Infrastructure management automation for æternity nodes
+
+## Ansible Deploy
+
+This implementation supports OpenStack dynamic inventory by using [Ansible 2.4 OpenStack inventory plugin](https://docs.ansible.com/ansible/devel/plugins/inventory/openstack.html).
+To install it's dependencies with `pip`:
+```bash
+pip install -r ansible/pip-requirements.txt
```

Comment:
pip-requirements.txt or requirements.txt ?

Reply:
fixed in efa9504c7fd927de7c20664571ab09b6fe037207